code/modules/vehicles/tonk.dm

```diff
@@ -0,0 +1,354 @@
+/*
+TONKS BY KMC
+THIS IS LIKE REGULAR CM TONK BUT IT'S LESS SHIT
+WHOEVER MADE CM TANKS: YOU ARE A BAD CODER!!!!!
+*/
+
+//generic component stuff
+/atom
+	var/tank_crash_damage = 10  //Arbitrary number. Set this to whatever you want with extra walltypes
```

Comment:
Could you add atom vars to `atoms.dm` ?

Reply:
I really dislike having to use this at all, damage from crashing into something should be based on a more generic value